Documentation/ceph-upgrade.md

```diff
@@ -221,6 +221,14 @@ Replace the namespace names in the new resources:
 sed "s/ROOK_SYSTEM_NAMESPACE/$ROOK_SYSTEM_NAMESPACE/g" upgrade-from-v1.0-create.yaml > upgrade-from-v1.0-create.yaml.tmp
 sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-create.yaml.tmp > upgrade-from-v1.0-create.yaml
 rm -f upgrade-from-v1.0-create.yaml.tmp
+sed -i "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-apply.yaml
```

Comment:
For consistency use the same sed as https://github.com/rook/rook/pull/3902/files#diff-3de4d43beba101be122197d34b432653R222

Reply:
@leseb are you suggesting to do below
```
sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-apply.yaml > upgrade-from-v1.0-apply.yaml.tmp
mv  upgrade-from-v1.0-create.yaml.tmp upgrade-from-v1.0-create.yaml
```

Reply:
Actually, I must be missing something, isn't your command exactly the same as L222? Why do you need `upgrade-from-v1.0-apply.yaml` if we have `upgrade-from-v1.0-create.yaml`?

Reply:
we have some changes in `upgrade-from-v1.0-apply.yaml file also, need to replace namespace and apply the role changes

Reply:
Ok, thanks, sounds good then.